docs/source/tagset/elemento-supplementary-material.rst

```diff
@@ -3,42 +3,37 @@
 <supplementary-material>
 ------------------------
 
-Aparece em
-  :ref:`elemento-article-meta`,
-  :ref:`elemento-p`,
+Aparece em:
+
+  :ref:`elemento-article-meta`
+  :ref:`elemento-p`
   :ref:`elemento-app`
 
-Atributos obrigatórios
-  1. id (ver :ref:`sugestao-atribuicao-id`)
-  2. xlink:href
-  3. mimetype
-  4. mime-subtype
- 
-Ocorre
+Atributos obrigatórios:
+
+  1. ``@id`` (ver :ref:`sugestao-atribuicao-id`)
+  2. ``@xlink:href``
+  3. ``@mimetype``
+  4. ``@mime-subtype``
+
+Ocorre:
+
   Zero ou mais vezes
 
 
-O material suplementar é um documento que não faz parte do texto do artigo, 
-mas que serviu como apoio para sua elaboração.
-Em ``<supplementary-material>`` é possível especificar tabelas, figuras, 
-dados brutos de planilha, banco de dados de genomas, quiz, equações, links, 
-diálogos, listas, licenças e objetos multimídia como áudio e vídeo.
- 
-O material suplementar pode estar em :ref:`elemento-front`, dentro de 
-:ref:`elemento-article-meta`, em :ref:`elemento-body` como seção ou entre 
-parágrafos ou em :ref:`elemento-back`, onde só poderá ser identificado caso 
-esteja especificado dentro do grupo de apêndices ``app-group``.
- 
-Seus atributos obrigatórios são:
- 
-* ``@id``: Utilizado como um identificador único no :term:`documento` e ganha maior 
-  importância quando há mais que um material suplementar e/ou quando o material 
-  suplementar é referenciado no corpo do texto. Nesse caso é necessário relacionar 
-  a chamada no texto com o "id" do material suplementar.
-* ``@mimetype``: Utilizado para especificar o tipo de mídia como "vídeo" ou "aplicação".
-* ``@mime-subtype``: Utilizado para especificar o formato da mídia.
-* ``@xlink:href``: Utilizado para indicar do nome completo do arquivo, tais como: pdf, vídeo, zip etc.
- 
+O material suplementar é um :term:`documento` que não faz parte do texto do artigo, mas que serviu como apoio para sua elaboração. Em ``<supplementary-material>`` é possível especificar tabelas, figuras, dados brutos de planilha, bancos de dados de genomas, quiz, equações, links, diálogos, listas, licenças e objetos multimídia como áudio e vídeo.
+
+Este elemento pode ser descrito em :ref:`elemento-front`, em :ref:`elemento-article-meta`, em :ref:`elemento-body` como seção ou entre parágrafos, ou em :ref:`elemento-back` onde só poderá ser identificado dentro do grupo de apêndices (``app-group``).
+
```

Comment:
Sim e não. É geralmente usado para definir um tipo de questionamento durante um jogo mental (palavras cruzadas, por exemplo). Se for algo de cunho científico deve provavelmente ser 'survey'.